cmd/lifecycle/analyzer.go

```diff
@@ -120,6 +120,30 @@ func (a *analyzeCmd) Privileges() error {
 		if err != nil {
 			return cmd.FailErr(err, "initialize docker client")
 		}
+	} else {
+		if a.platformAPIVersionGreaterThan06() {
+			err := a.verifyReadAccess(a.previousImage)
+			if err != nil {
+				return cmd.FailErr(err)
+			}
+			err = a.verifyReadAccess(a.runImageRef)
+			if err != nil {
+				return cmd.FailErr(err)
+			}
+			if len(a.additionalTags) > 0 {
+				for _, tag := range a.additionalTags {
+					err := a.verifyReadWriteAccess(tag)
+					if err != nil {
+						return cmd.FailErr(err)
+					}
+				}
+			}
+		} else {
```

Comment:
Please correct me if I'm wrong but this is a new validation for platform API <= 0.6, right?
Does it mean that a user that is already successfully using an old platform API version will start have failures after upgrading to the new lifecycle version?